subscription-manager.spec

```
@@ -1333,6 +1333,15 @@ find %{buildroot} -name \*.py* -exec touch -r %{SOURCE0} '{}' \;
     %endif
 %endif
 
+# When subscription-manager is upgraded on RHEL 8 (from RHEL 8.2 to RHEL 8.3), then kill
+# instance of rhsmd, because it is not necessary anymore and it can cause issues.
+# See: https://bugzilla.redhat.com/show_bug.cgi?id=1840364
+%if ( 0%{?rhel} >= 8 || 0%{?fedora} )
+if [ "$1" = "2" ] ; then
+    killall rhsmd
```

We should also add the "| true" here to make sure that a falure to find and kill rhsmd processes (which may not be running during install time) is not considered a failure.
```suggestion
    killall rhsmd | true
```

---

You are right. I also redirected to `stderr` to `/dev/null` of `killall` command. I force updated the PR.